src/Directory.Build.props

```diff
@@ -1,7 +1,3 @@
 <Project>
   <Import Project="..\Directory.Build.props" />
```

Comment:
This file can be removed. It doesn't provide any value.

Reply:
@pgovind beware that if you do endup removing this file, then you will need to update all of the projects that were importing this in order to import the root props file.